api/schedule/update.go

```diff
@@ -127,6 +127,7 @@ func UpdateSchedule(c *gin.Context) {
 
 	// set the updated by field using claims
 	s.SetUpdatedBy(u.GetName())
+	s.SetBranch(input.GetBranch())
```

Comment:
We'll want to follow the same pattern as above in order to make sure an update payload like this: 
```json
{
    "entry": "0 0 * * *"
}
```
Doesn't result in the branch being set to an empty string.

```go
if input.GetBranch() != "" {
    s.SetBranch(input.GetBranch)
}
```

Reply:
I updated the code to reflect that